l10n_it_vat_registries/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,42 @@
+**Italiano**
+
+E' possibile configurare quali imposte escludere dai registri (ad esempio le ritenute)
+impostando il campo dell'imposta 'Escludi dai registri IVA'
+
+Per generare i periodi fiscali,
+aprire Contabilità > Configurazione > Contabilità > Intervalli date > Genera intervalli date.
+
+* Prefisso: Prefisso identificativo dell'anno dei periodi da generare
+* Durata: 1 mese
+* Numero di intervalli da generare: 12
+* Tipo: Creare un tipo o utilizzarne uno esistente, non è richiesta una configurazione particolare per il tipo
+* Data di inizio: primo giorno dell'anno dei periodi da generare (ad esempio 01/01/2018)
+
+Tramite
+Contabilità -> Configurazione -> Contabilità -> Registri IVA
+è possibile configurare i registri IVA, ad esempio 'vendite', 'acquisti' o 'corrispettivi'.
+Per associare un registro ad un registro IVA, aprire il registro tramite
+Contabilità -> Configurazione -> Contabilità -> Registri
+ed impostare il campo 'Registro IVA'
```

Comment:
Come da linee guida, di solito in inglese si usano gli apici singoli ' ', in italiano quelli doppi "  " o quelli bassi «  ».
Per evitare il doppio termine "registro" ravvicinato:
Per associare un registro contabile ad/a un registro IVA, ...

ed/e impostare

P.S.: riguardo a/ad e/ed, sembra sia convenzione non usare la d se dopo c'è una vocale diversa.
http://dizionari.corriere.it/dizionario-si-dice/A/a-ad.shtml
Ma è una finezza.